### PR TITLE
chore: add gh action to release helm chart as a gh-page

### PR DESCRIPTION
**Pull request by tiagolobocastro**
Adds the release workflow and set the mayastor version to 2.0.0-develop.

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -22,9 +22,8 @@ version: 1.0.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.0.0"
+appVersion: "2.0.0-develop"
```
Comment (vharsh) [inline chart/Chart.yaml:25]: will have to think about also keeping older Helm chart releases

I'd need to take a look at how it's done for OpenEBS

**Review by vharsh — APPROVED** [review]
lgtm

**Review by Abhinandan-Purkait — APPROVED** [review]

File: chart/Chart.yaml (hunk 1 above)
Comment (tiagolobocastro) [reply chart/Chart.yaml:25]: Yep, would be good to have a plan. Also would we want to include this in the OpenEBS charts?

**Comment by tiagolobocastro** [comment]
bors merge

File: chart/Chart.yaml (hunk 1 above)
Comment (vharsh) [reply chart/Chart.yaml:25]: I'll put this up for discussion in the community meeting, if the answer is yes, it'd become another sub Helm chart under OpenEBS chart.

**Pull request merged**